explorer/idl/explorer_v2.idl

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2019 IoTeX
```

Comment:
Should move this to another package, with a different name, e.g., api/idl/api_v2.idl?

Reply:
Yes.